AsyncDisplayKit/ASDisplayNodeExtras.h

```diff
@@ -82,24 +82,24 @@ extern ASDisplayNode * _Nullable ASViewToDisplayNode(UIView * _Nullable view);
 extern ASDisplayNode *ASDisplayNodeUltimateParentOfNode(ASDisplayNode *node);
 
 /**
- This function will walk the layer hierarchy, spanning discontinuous sections of the node hierarchy (e.g. the layers
- of UIKit intermediate views in UIViewControllers, UITableView, UICollectionView).
+ If traverseSublayers == YES, this function will walk the layer hierarchy, spanning discontinuous sections of the node hierarchy\
+ (e.g. the layers of UIKit intermediate views in UIViewControllers, UITableView, UICollectionView).
  In the event that a node's backing layer is not created yet, the function will only walk the direct subnodes instead
  of forcing the layer hierarchy to be created.
  */
```

Comment:
Should we add to the documentation that if `traversingSublayers = NO` we still try to talk the node hierarchy?